test/exportToNDJson.test.js

```diff
@@ -30,6 +31,10 @@ describe('check export logic', () => {
     expect(fs.existsSync(expectedFileName)).toBe(true);
   });
 
+  test('Expect folder created and export successful when _typeFilter  parameter is retrieved from request', async () => {
+    await exportToNDJson(clientId, mockType, mockTypeFilter);
+    expect(fs.existsSync(expectedFileName)).toBe(true);
+  });
```

Comment:
Can we do a test with multiple queries/resources/properties/urls as part of the filter? Could probably do multiples of all of those at once in a single filter. Seems like there's a bit of complexity with managing all of those at the same time, so it would be good to have a test for that.